src/Libraries/RevitNodesUI/RevitDropDown.cs

```diff
@@ -478,7 +478,10 @@ protected override SelectionState PopulateItemsCore(string currentSelection)
                 if (category != null)
                 {
                     string name = getFullName(category);
-                    Items.Add(new DynamoDropDownItem(name, categoryId));
+                    if (name != "Cameras")
```

Comment:
I might be wrong on this, but I can't for love of god find an object that you could assign OST_Cameras category to. I thought things like Security Devices would be allowed to have that category but its not available. Perhaps something like a NodeView or a CameraInfo object is of that category...I don't know. I got misled by Marcello trying to use that category to collect "Camera Views" which in reality are just View3D objects but that's not a great reason to hide it. I will reverse these changes.